events/open-source-marathon.html

```diff
@@ -31,6 +31,183 @@
         content: "-";
         width: 1.5rem;
       }
+/*styles of glitch effect*/
+      .glitch-wrapper {
+   width: 100%;
+   height: 100%;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+   text-align: center;
+  
+}
+
+.glitch {
+   position: relative;
+   font-size: 3.75rem;
+   font-weight: bold;
+   color: #FFFFFF;
+   letter-spacing: 3px;
+   z-index: 1;
+}
+
+.glitch:before {
+   content: attr(data-text);
+   position: absolute;
+   top: 0;
+   left: -2px;
+   width: 100%;
+   color: 50px;
+   background-image: url("/assets/img/grain-dark.png");
```

Comment:
What about `color` having 50px property?
I think `background-image` is not necessary here, you can just remove it.